gensim/models/atmodel.py

```diff
@@ -326,11 +336,20 @@ def init_empty_corpus(self):
             self.corpus = []
 
     def extend_corpus(self, corpus):
-        """
-        Add new documents in `corpus` to `self.corpus`. If serialization is used,
-        then the entire corpus (`self.corpus`) is re-serialized and the new documents
-        are added in the process. If serialization is not used, the corpus, as a list
-        of documents, is simply extended.
+        """Add new documents from `corpus` to `self.corpus`.
+
+        If serialization is used, then the entire corpus (`self.corpus`) is re-serialized and the new documents
+        are added in the process. If serialization is not used, the corpus, as a list of documents, is simply extended.
+
+        Parameters
+        ----------
+        corpus : iterable of list of (int, float)
+            Corpus in BoW format
+
+        Raises
+        ------
+        AssertionError
```

Comment:
`AssertionError` is for programmer mistakes. Bad user input is either `ValueError` or `TypeError`.

Reply:
Question for the author of this code, not for me.